Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -286,6 +287,24 @@ open class AVFoundationPlayback: Playback {
             selector: #selector(AVFoundationPlayback.playbackDidEnd),
             name: NSNotification.Name.AVPlayerItemDidPlayToEndTime,
             object: player.currentItem)
+        
+        NotificationCenter.default.addObserver(
```

Comment:
Since bitrate is an "agnostic" information regardless of the playback type, does it make sense to move the bitrate tracking responsibility out of the `AVFoundationPlayback`? 

The benefits I can point out so far:
1. One less responsibility for the AVFoundationPlayback class
2. `.didUpdateBitrate` event wouldn't be coupled to `AVFoundationPlayback`
3. Other playbacks could benefit from this bitrate event as well

I was thinking if the `Playback` class could play this role instead. 
The only thing we should create an abstraction for is the notification mechanism.